app/backend/api.ts

```diff
@@ -56,3 +57,12 @@ export const getPaymentDetails = async (payeeName: string, partPostcode: string)
 
 	return JSON.parse(response.payload)
 }
+
+
+export const getDownloadDetailsCsv = async (payeeName: string, partPostcode: string): Promise<Buffer> => {
+	const url = getUrlParams('downloaddetails', {
+		payeeName,
+		partPostcode
+	})
+	return utils.getBuffer(`${config.backendEndpoint}${url}`)
```

Comment:
I have just realised that getBuffer is actually defined in the wrong place. Similar to get and post methods, getbuffer should be defined within the backend folder, so ideally here in the api.ts file because it calls the BE service.
I would also suggest adding a try catch block within getBuffer to catch any exceptions here rather than in the handler, so any other subsequent calls to this function has got error handling in place. 
Within the handler itself, you can check for a null or any other default value you want to return from getDownloadDetailsCsv and then return an appropriate error back to FE instead of throwing an error. Hope that makes sense.
Also, we need some unit tests to cover getBuffer and getDownloadDetailsCSV.